### ISSUE TITLE
Appli not shuting down (v0.9.8 - BTC_DAO_REGTEST mode)

### ISSUE THREAD

**Issue by ghost** [bisq-network/bisq#2673]
Appli not shuting down (v0.9.8 - BTC_DAO_REGTEST mode)
Debian stretch 9.8.
Had to force quit.
Doesn't seem to be a great issue (seems related to previous similar issues) but I prefer to mention it.
Last lines in CLI:
<pre>
...
avr.-08 19:41:41.439 [JavaFX Application Thread] INFO  b.c.p.price.PriceFeedService: Received new MarketPrice(currencyCode=EUR, price=4633.57, timestampSec=1554745418226, isExternallyProvidedPrice=true) from provider http://44mgyoe2b6oqiytt.onion/ after 0.772 sec. 
avr.-08 19:42:02.023 [InputHandler-opkdqqo2xozic6hk.onion:9999] INFO  b.n.p2p.network.Connection: proto is null because protoInputStream.read()=-1 (EOF). That is expected if client got stopped without proper shutdown. 
avr.-08 19:42:02.023 [InputHandler-t467j4bfbyqj5tqv.onion:9999] INFO  b.n.p2p.network.Connection: proto is null because protoInputStream.read()=-1 (EOF). That is expected if client got stopped without proper shutdown. 
avr.-08 19:42:02.023 [InputHandler-mpbvohr3xccy7bzc.onion:9999] INFO  b.n.p2p.network.Connection: proto is null because protoInputStream.read()=-1 (EOF). That is expected if client got stopped without proper shutdown. 
avr.-08 19:42:02.025 [InputHandler-b3jnw7fyph2jsu6n.onion:8005] INFO  b.n.p2p.network.Connection: proto is null because protoInputStream.read()=-1 (EOF). That is expected if client got stopped without proper shutdown. 
avr.-08 19:42:02.026 [InputHandler-2bnvhfkdrnx5hrlv.onion:8005] INFO  b.n.p2p.network.Connection: proto is null because protoInputStream.read()=-1 (EOF). That is expected if client got stopped without proper shutdown. 
avr.-08 19:42:02.026 [InputHandler-rvtp3f5teuinofyc.onion:9999] INFO  b.n.p2p.network.Connection: proto is null because protoInputStream.read()=-1 (EOF). That is expected if client got stopped without proper shutdown. 
avr.-08 19:42:02.027 [InputHandler-wzkti35unlo3fyny.onion:9999] INFO  b.n.p2p.network.Connection: proto is null because protoInputStream.read()=-1 (EOF). That is expected if client got stopped without proper shutdown. 
avr.-08 19:42:02.027 [InputHandler-3ddk5ge5lykzhrmg.onion:9999] INFO  b.n.p2p.network.Connection: proto is null because protoInputStream.read()=-1 (EOF). That is expected if client got stopped without proper shutdown. 
avr.-08 19:45:43.456 [Wallet autosave thread] INFO  o.bitcoinj.wallet.WalletFiles: Background saving wallet; last seen block is height 1931, date 2019-04-08T17:45:38Z, hash 2c5e07b110c8e7fbcb60040e8da996d244882cb2eb9a37ae810e01c4bd287c9c 
avr.-08 19:45:43.456 [Wallet autosave thread] INFO  o.bitcoinj.wallet.WalletFiles: Background saving wallet; last seen block is height 1931, date 2019-04-08T17:45:38Z, hash 2c5e07b110c8e7fbcb60040e8da996d244882cb2eb9a37ae810e01c4bd287c9c 
avr.-08 19:45:43.539 [Wallet autosave thread] INFO  o.bitcoinj.wallet.WalletFiles: Save completed in 82.82 ms 
avr.-08 19:45:43.539 [Wallet autosave thread] INFO  o.bitcoinj.wallet.WalletFiles: Save completed in 82.66 ms 
Processus arrêté
</pre>

**Comment by ManfredKarrer** [comment]
Have you seen that with the current mainnet version (non dao) as well? I saw it also a few times recently but as I test with DAO usually it was the DAO enabled version. Would be just good to see if it is related to DAO code or not. 
I suspect a out or heap memory issue. Can you check for OS logs if you find anything?

**Comment by ghost** [comment]
Indeed, in https://github.com/bisq-network/bisq/issues/2634 I did several shutdowns in MAINNET and BTC_DAO_REGTEST mode, and it only happened in BTC_DAO_REGTEST mode.

in syslog.1, at the time the issue happened:
<pre>
Apr  8 19:41:08 DO790C gnome-shell[939]: STACK_OP_LOWER_BELOW: window 0x2200007 (JavaEmbedd) not in stack
idem
Apr  8 19:41:37 DO790C gnome-shell[939]: STACK_OP_RAISE_ABOVE: window 0x2200007 (JavaEmbedd) not in stack
idem
Apr  8 19:41:41 DO790C gnome-shell[939]: STACK_OP_LOWER_BELOW: window 0x2200007 (JavaEmbedd) not in stack
Apr  8 19:41:41 DO790C gnome-shell[939]: STACK_OP_RAISE_ABOVE: window 0x2200007 (JavaEmbedd) not in stack
Apr  8 19:41:41 DO790C gnome-shell[939]: STACK_OP_LOWER_BELOW: window 0x2200007 (JavaEmbedd) not in stack
Apr  8 19:41:43 DO790C gnome-shell[939]: STACK_OP_LOWER_BELOW: window 0x2200007 (JavaEmbedd) not in stack
Apr  8 19:41:47 DO790C tracker-extract[1131]: unable to create file '/run/user/1000/dconf/user': Permission non accordée.  dconf will not work properly.
Apr  8 19:41:47 DO790C gnome-shell[939]: STACK_OP_RAISE_ABOVE: window 0x2200007 (JavaEmbedd) not in stack
idem
Apr  8 19:41:57 DO790C gnome-shell[939]: STACK_OP_LOWER_BELOW: window 0x2200007 (JavaEmbedd) not in stack
Apr  8 19:41:57 DO790C gnome-shell[939]: STACK_OP_RAISE_ABOVE: window 0x2200007 (JavaEmbedd) not in stack
Apr  8 19:41:57 DO790C gnome-shell[939]: STACK_OP_LOWER_BELOW: window 0x2200007 (JavaEmbedd) not in stack
idem
Apr  8 19:43:04 DO790C gnome-terminal-[1085]: Allocating size to GtkScrollbar 0x562a6d8d42c0 without calling gtk_widget_get_preferred_width/height(). How does the code know the size to allocate?
...
Apr  8 19:44:52 DO790C gnome-shell[939]: STACK_OP_LOWER_BELOW: window 0x2200007 (JavaEmbedd) not in stack
idem
Apr  8 19:48:21 DO790C zenity[3206]: GtkDialog mapped without a transient parent. This is discouraged.
</pre>
Same messages in other log files.

**Comment by Giszmo** [comment]
Had to force-quit here, too. v1.1.2 debian.

**Comment by HenrikJannsen** [comment]
Fixed in 1.10.6
